modules/installation-disk-partitioning.adoc

```diff
@@ -24,6 +24,11 @@ officially supported for making `/var` or a subdirectory of `/var`, such as `/va
 +
 [IMPORTANT]
 ====
+For disk sizes larger than 100G, and especially disk sizes larger than 1TB, it is strongly recommended to create a separate `/var` partition. See "Creating a separate `/var` partition" for more information.
+====
++
+[IMPORTANT]
+====
 Kubernetes supports only two filesystem partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
```

Comment:
"Filesystem" is two words.
[
](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f)<img width="185" alt="image" src="https://user-images.githubusercontent.com/83772007/168335184-0c82504e-c461-4c36-8d5b-eb98509fc21e.png">

Reply:
I can change this too, though it's technically outside the scope of my PR changes 😄